ci/dummy_scenario.erl

```diff
@@ -43,8 +43,8 @@ test_verification_function(_)    -> {true, new_value}.
 
 -spec init() -> ok.
 init() ->
-    ok = amoc_metrics:init(counters, dummy_scenario_metric),
-    ok = amoc_metrics:init(times, dummy_scenario_metric),
```

Comment:
Wouldn't this still fail? After some digging (IMO it should be better documented) I found out that `declare` returns the result of `ets:insert_new`. If the key already exists, `false` is returned.

Reply:
Yeah, but that's just running on CI, where we expected the metric not to exist.

Also, for example for gauges, it says it'll return false if the metric already existing https://hexdocs.pm/prometheus/prometheus_gauge.html#declare/1 :)

Reply:
Ok, I thought of this scenario as an example, and if someone copies this into a real scenario, it would stop working. But it's minor, and OK to merge for me.